examples/distributed_stencil/1d_stencil.cpp

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) 2020 Nikunj Gupta
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
```

Comment:
You could add a short description here saying what this does differently than the other stencil examples (something like this: https://github.com/STEllAR-GROUP/hpx/blob/6172cea92789ab5e30aa815673418aebbbd04456/examples/1d_stencil/1d_stencil_8.cpp#L7-L10).